src/components/Chips/Chips.tsx

```diff
@@ -110,14 +112,35 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
     const overrideDataTestId = dataTestId || getTestId(ComponentDefaultTestId.CHIP, id);
     const hasClickableWrapper = (!!onClick || !!onMouseDown) && !disableClickableBehavior;
     const hasCloseButton = !readOnly && !disabled;
-
-    const focusRef = useRef(null);
-    const [hoverRef, isHovered] = useHover();
-    const { isFocused } = useSetFocus({ ref: focusRef });
+    const overrideAriaLabel = ariaLabel || label ? ariaLabel || label : undefined;
```

Comment:
a bit confusing, why it's not: 
ariaLabel || label;

Reply:
It's just always will equal empty string, even if ariaLabel and label are not passed - due to label default value.. maybe it's nothing, just don't like seeing this empty attributes in each snapshot. wdyt?